test/e2e/plugin_sync/tmc/plugin_sync_tmc_lifecycle_test.go

```diff
@@ -666,7 +666,183 @@ var _ = f.CLICoreDescribe("[Tests:E2E][Feature:Plugin-Sync-TMC-lifecycle]", func
 		})
 	})
 
-	// Use Case 7: Plugin List, sync, search and install functionalities with Context Issues
+	// Use case 7: Sync for single target specific plugins, and validate the plugin list
+	// run context create (make sure another target context is active, but yet to install plugins), it should not perform the sync for all active contexts
+	// run target specific plugin sync (for k8s target), make sync should not happen for tmc context even though its active
+	// run target specific plugin sync (for tmc target), make sync should not happen for k8s context even though its active
```

Comment:
Comments to be updated to say `contextType` instead of `target` depending on what is decided about the flag name for `tanzu plugin sync`